### PR TITLE
Add website link for tbhaxor

### PR DESCRIPTION
**Pull request by tbhaxor**

### CODE REVIEW

File: people.json [hunk 1]
```diff
@@ -340,8 +340,10 @@
     "links": {
       "GitHub": "https://github.com/tbhaxor",
       "Twitter": "https://twitter.com/tbhaxor",
-      "LinkedIn": "https://linkedin.com/in/tbhaxor"
+      "LinkedIn": "https://linkedin.com/in/tbhaxor",
+      "Website": "https://tbhaxor.com/?source=serenity",
```
Comment (linusg) [inline people.json:344]:
```suggestion
      "Website": "https://tbhaxor.com",
```

Use referrer-based tracking on your end if you need that.

File: people.json [hunk 2]
```diff
@@ -340,8 +340,10 @@
...
     }
+    "contributor": true
```
Comment (linusg) [inline people.json:346]: AFAICT you have made [one commit](https://github.com/SerenityOS/serenity/commits?author=tbhaxor) about a year ago replacing a total of three words? This seems like a stretch...

**Review by linusg — CHANGES_REQUESTED** [review]

File: people.json (hunk 2 above)
Comment (tbhaxor) [reply people.json:346]: Cool I will be updating it.

Comment (tbhaxor) [reply people.json]: @linusg Fixed

**Pull request merged**